python/pythonlab/neighborhood/neighborhood/painter.py

```diff
@@ -124,28 +161,28 @@ def is_facing_north(self):
     Returns:
       True if the painter is facing North
     """
-    return False
+    return self.direction.value == 'north'
```

Comment:
we can use `self.direction.is_north()`, etc. for these methods

Reply:
I was considering adding a getter for the `Direction`'s `value`, but we don't do this for the `World`'s grid so leaving as-is.